waspc/data/Generator/templates/sdk/wasp/auth/useAuth.ts

```diff
@@ -20,6 +20,7 @@ function createUserGetter() {
   async function getMe(): Promise<AuthUser | null> {
     try {
       const response = await api.get(getMeRoute.path)
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
I got to remove this, this is just while testing so I can see the loader nicely.